numpy/distutils/exec_command.py

```diff
@@ -249,11 +250,13 @@ def _exec_command(command, use_shell=None, use_tee = None, **env):
         proc = subprocess.Popen(command, shell=use_shell, env=env,
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.STDOUT,
-                                universal_newlines=True)
+                                universal_newlines=False)
     except EnvironmentError:
         # Return 127, as os.spawn*() and /bin/sh do
         return 127, ''
     text, err = proc.communicate()
+    text = text.decode(locale.getpreferredencoding(False),
```

Comment:
Should this follow Python's default i/o encoding? sys.getdefaultencoding()?

Reply:
I think this has to be compatible with the stdout encoding, so that you don't get failures later on when trying to print this stuff.

Reply:
Thanks I'll try that.